README.md

```diff
@@ -150,8 +150,34 @@ and set nozeroconf to True (no 169.254.0.0/16 link local address).
 	            address: 192.168.20.18
 	            netmask: 255.255.255.0
 ```
+6) Configure Bond interface over vlan tagged 10001 interfaces with extra route.
+important part is define that slaves for the bond are vlan slaves
+and bond carrier detection is not using physical interface link status. This feature is extremely usable whenever one needs better utilization of physical interfaces. two physical interfaces can support 2 or more different bonded interfaces  that have different tagged vlans. This means that for example public and private networks ( 2 different tagged vlans) can be separated  to different physical interfaces for communication and in failure state, fail over to available interface. 
 
-6) All the above examples show how to configure a single host, The below
+```
+- hosts: myhost
+  roles:
+    - role: network
+      network_extra_bonding_module_options: "use_carrier=0"
+      network_bond_interfaces:
+        - device: bond-ext
+          address: 192.168.10.128
+          netmask: 255.255.255.0
+          bootproto: static
+          bond_mode: active-backup
+          onboot: "yes"
+          nm_controlled: "no"
+          mtu: 9000
+          bond_miimon: 500
+          bond_slaves: [eth1.10001, eth2.10001]
```

Comment:
10001 is a typo here and in the description, right? Plz 2 fix

Reply:
I'm guessing that junosia is referring to that VLAN numbers can't be over 4096.

Reply:
It is just an example. If it less than 4096, it might break production vlans?

Reply:
Yes but it's not a VLAN since it won't fit into the 12 bit VLAN tag field, creating confusion for the reader. If you want to exclude VLANs in use, any VLAN outside the ones mentioned in https://wiki.csc.fi/CloudComputing/PoutaNetworkAddresses is fine